src/main/resources/docs/description/description.json

```diff
@@ -1,255 +1,290 @@
 [
   {
-    "patternId":"B101",
-    "title":"Assert used"
+    "patternId" : "B101",
+    "title" : "assert_used"
```

Comment:
I'm also feeling unsure about these title changes

Reply:
I think having better titles is fine, the patternId are not changing.